fairlearn/postprocessing/_roc_curve_utilities.py

```diff
@@ -8,11 +8,21 @@
 from ._threshold_operation import ThresholdOperation
 
 
-def _get_roc(data, x_grid, attribute, flip=True):
+def _get_roc(data, sensitive_feature_value, flip=True):
     """Get ROC curve's convex hull based on data columns 'score' and 'label'
     Scores represent output values from the predictor.
+
+    :param data: the DataFrame containing scores and labels
+    :type data: pandas.DataFrame
```

Comment:
Side note, with an update to `conf.py` we should be able to have things like this link to the Pandas documentation (like we already can for `sklearn`). In part, the task I created for documentation consistency is about things like that.

Reply:
what's conf.py?

Reply:
The file which configures sphinx. It's in the docs directory. The above is more a note for the future, and needn't delay this PR.

Reply:
OK great!